json-2/src/main/java/com/baeldung/jsonld/serialization/jacksonjsonld/JacksonJsonldExamplePerson.java

```diff
@@ -0,0 +1,17 @@
+package com.baeldung.jsonld.serialization.jacksonjsonld;
+
+import ioinformarics.oss.jackson.module.jsonld.annotation.JsonldId;
+import ioinformarics.oss.jackson.module.jsonld.annotation.JsonldLink;
+import ioinformarics.oss.jackson.module.jsonld.annotation.JsonldNamespace;
+import ioinformarics.oss.jackson.module.jsonld.annotation.JsonldProperty;
+import ioinformarics.oss.jackson.module.jsonld.annotation.JsonldType;
+
+@JsonldNamespace(name = "s", uri = "http://schema.org/")
+@JsonldLink(rel = "s:knows", name = "knows", href = "http://example.com/person/2345")
+@JsonldType("s:Person")
+class JacksonJsonldExamplePerson {
```

Comment:
As these POJO classes are already in a separate package, can we not simply call the class Person?